lib/prompt.js

```diff
@@ -85,7 +86,11 @@ function prompt (name, options) {
   let spinnerTask
   if (options.mask || options.hide) {
     if (!isTTY) {
-      return Promise.reject(`CLI needs to prompt for ${options.name || options.prompt} but stdin is not a tty.`)
+      let errMsg = `CLI needs to prompt for ${options.name || options.prompt} but stdin is not a tty.`
+      if (os.platform() === 'win32') {
+        errMsg += ' This is likely a problem with your shell sofware. On win32 plaftorms, we recommend using cmd.exe'
```

Comment:
@KarateCowboy What would you think of dropping `This is likely a problem with your shell sofware.` and changing `win32` to `Windows`?